src/BoundaryConditions/BoundaryConditions.jl

```diff
@@ -41,5 +41,5 @@ include("compute_flux_bcs.jl")
 include("update_boundary_conditions.jl")
 include("polar_boundary_condition.jl")
 
-include("perturbation_advection_open_boundary_scheme.jl")
+include("perturbation_advection_open_scheme.jl")
```

Comment:
```suggestion
include("perturbation_advection.jl")
```

this is the convention that a file which introduces a type is named after the type

Reply:
Sounds good! Will make the change soon